### ISSUE TITLE
Domain, e-mail and website management (Microsoft 365?)

### ISSUE THREAD

**Issue by Piedone** [OrchardCMS/OrchardCore#6254]
We need shared management of the domains and e-mails of the Orchard ecosystem to allow:
- Multiple people having administrator access so nobody is a single point of failure, or has to do a certain job alone all the time.
- Creating e-mail groups. Notably, we need at least two central e-mail addresses that are forwarding to multiple people: A general one that can be used in places like Cloudsmith (see https://github.com/OrchardCMS/OrchardCore/issues/6238) and one for security matters, e.g. security@orchardcore.net.

Furthermore, these and the website hosting is currently solely the task and financial responsibility of @agriffard.

Can .NET Foundation provide something for these? Maybe a Microsoft 365 tenant and Azure credits? I'd assume there are established practices they provide for at least the administrative part of this (and e.g. managing credentials), perhaps also online services like these?

As an overarching goal, so we can grow as a community, and as a project I think we should get rid of every single point of failure and every important knowledge being stuck in somebody's head. So have at least two people as administrators everywhere, and document important procedures (hence my effort under https://github.com/OrchardCMS/OrchardCore/pull/6119). This not just adds redundancy but this way responsibility can be spread out more evenly too (if it's only you who knows how to do something then you'll do it every time), as well as open up new avenues of possible contributions.

Another such issue I see is with the packages on NuGet (Sebastien being the sole owner).

**Comment by sebastienros** [comment]
I will submit the issue to DNF.

**Comment by Piedone** [comment]
Thanks!

**Comment by Piedone** [comment]
Any news here @sebastienros?

**Comment by sebastienros** [comment]
What's left from this issue? I believe we have multiple administrators on the repos, and the NuGet organization is now working well. Same for cloudsmith. Certificates are also automatically handled (by Azure). So only being able to have email addresses? Probably also the Azure subscription that hosts the website.

**Comment by Piedone** [comment]
Yes, e-mails and Azure. I believe this is what you wanted to get in touch with DNF about.
